.github/codeql/codeql-config.yml

```diff
@@ -0,0 +1,4 @@
+paths:
+  - src
+  - profiles
+  - js
```

Comment:
Maybe this instead? This will include tools and worker too.
```suggestion
paths-ignore:
  - builds
  - tests
```

Reply:
Good point.